docs/source/dev_guide/project-standards.rst

```diff
@@ -93,17 +93,17 @@ How to squash and rebase commits
 
 Assuming that you followed :ref:`"(b) Development Environment" <dev-env>`:
 
-1. Sync ``master`` with upstream ``master`` ::
+1. Sync ``master`` with the main repo's ``master`` ::
 
     git checkout master
-    git rebase upstream/master
-    git push -f origin master
+    git rebase <upstream-origin>/master
+    git push -f <fork-origin> master
 
 2. Rebase branch onto ``master`` ::
 
-    git checkout branchName
-    git rebase master
-    git push -f origin branchName
+    git checkout <branch-name>
+    git rebase master # If you have merge conflicts, it may be easier to `git rebase --abort` and first do step 4 with SHA of the last commit before your work
```

Comment:
@tomvothecoder I'll just leave this line in here until one of us changes the order of the steps in a future pull request.

Reply:
#418 will update this